DuckDuckGo/TabBar/ViewModel/TabCollectionViewModel.swift

```diff
@@ -89,6 +89,17 @@ final class TabCollectionViewModel: NSObject {
         didSet {
             previouslySelectedTabViewModel = oldValue
             oldValue?.tab.renderTabSnapshot()
+
+            if #available(macOS 14.4, *) {
+                if let oldValue {
+                    WebExtensionManager.shared.eventsListener.didDeselectTabs([oldValue.tab])
+                }
+                if let selectedTabViewModel {
+                    WebExtensionManager.shared.eventsListener.didSelectTabs([selectedTabViewModel.tab])
```

Comment:
will it (and should it?) fire at initial tab activation?

Reply:
I think it isn't necessary to call at initial tab activation since the engine calls `isSelected` from `_WKWebExtensionTab`.